### PR TITLE
feat: support heroku redis

### LINKED ISSUE #27: add support heroku redis
```
2016-10-12T09:15:29+00:00 app[heroku-redis]: source=REDIS sample#active-connections=3 sample#load-avg-1m=0.005 sample#load-avg-5m=0.035 sample#load-avg-15m=0.065 sample#read-iops=0 sample#write-iops=0 sample#memory-total=15664876.0kB sample#memory-free=13038516.0kB sample#memory-cached=646352kB sample#memory-redis=476168bytes sample#hit-rate=0.99048 sample#evicted-keys=0
```

### PR DESCRIPTION
**Pull request by proudlygeek**
Fix #27

Let me know if this works out for you or if you require changes :) it's my first golang PR so feel free to point things out, thanks 🙏

### CODE REVIEW

File: logproc.go [hunk 1]
```diff
@@ -113,18 +113,23 @@ func logProcess(in chan *logData, out chan *logMetrics) {
 		}
 
 		log.Debugln(*data.line)
-		output := strings.Split(*data.line, " - ")
+		var output []string
```
Comment (abtris) [inline logproc.go]: Not sure why you change this. `:=` was enough.

File: logproc.go [hunk 2]
```diff
@@ -113,18 +113,23 @@ func logProcess(in chan *logData, out chan *logMetrics) {
...
+		output = strings.Split(*data.line, " - ")
+		if len(output) < 2 {
+			output = strings.Split(*data.line, " app[heroku-redis]: ")
+		}
 		if len(output) < 2 {
 			continue
 		}
```
Comment (abtris) [inline logproc.go:120]: This looks as duplicity. That cause worse readability. Make change naming for getting redis parts and not reuse `output` variable will be better for this case. 

@w-vi what do you think?

File: logproc.go (hunk 1 above)
Comment (proudlygeek) [reply logproc.go]: Indeed, my bad 👍

File: logproc.go (hunk 2 above)
Comment (proudlygeek) [reply logproc.go:120]: I see your point 👍 let me try with an explicit variable to get the redis bits!

**Comment by abtris** [comment]
Thanks for contribution!

**Pull request merged**